lib/system/external.toit

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2024 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import io
+import rpc
+
+class ExternalMessageHandler_ implements SystemMessageHandler_:
+  static TYPE ::= SYSTEM-EXTERNAL-NOTIFICATION_
+  handler/Lambda := ?
+
+  constructor .handler:
+    set-system-message-handler_ TYPE this
+
+  close -> none:
+    clear-system-message-handler_ TYPE
```

Comment:
Should we clear the handler here? It would have to be nullable, which may be annoying.